2021-05-19 17:01:30,178 INFO    MainThread:12744 [wandb_setup.py:_flush():69] setting env: {}
2021-05-19 17:01:30,178 INFO    MainThread:12744 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-19 17:01:30,179 INFO    MainThread:12744 [wandb_init.py:_log_setup():336] Logging user logs to C:\Users\Lorenzo\PycharmProjects\flow-forecast\notebooks\wandb\run-20210519_170130-ys0ubt2f\logs\debug.log
2021-05-19 17:01:30,180 INFO    MainThread:12744 [wandb_init.py:_log_setup():337] Logging internal logs to C:\Users\Lorenzo\PycharmProjects\flow-forecast\notebooks\wandb\run-20210519_170130-ys0ubt2f\logs\debug-internal.log
2021-05-19 17:01:30,181 INFO    MainThread:12744 [wandb_init.py:_jupyter_setup():287] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000245B99BB408>
2021-05-19 17:01:30,184 INFO    MainThread:12744 [wandb_init.py:init():369] calling init triggers
2021-05-19 17:01:30,184 INFO    MainThread:12744 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-19 17:01:30,184 INFO    MainThread:12744 [wandb_init.py:init():418] starting backend
2021-05-19 17:01:30,185 INFO    MainThread:12744 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=spawn, using: spawn
2021-05-19 17:01:30,191 INFO    MainThread:12744 [backend.py:ensure_launched():123] starting backend process...
2021-05-19 17:01:30,298 INFO    MainThread:12744 [backend.py:ensure_launched():128] started backend process with pid: 9296
2021-05-19 17:01:30,299 INFO    MainThread:12744 [wandb_init.py:init():423] backend started and connected
2021-05-19 17:01:30,301 INFO    MainThread:12744 [wandb_init.py:init():465] updated telemetry
2021-05-19 17:01:30,301 INFO    MainThread:12744 [wandb_init.py:init():484] communicating current version
2021-05-19 17:01:31,840 INFO    MainThread:12744 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.30 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-19 17:01:31,844 INFO    MainThread:12744 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-19 17:01:34,044 INFO    MainThread:12744 [wandb_init.py:init():522] starting run threads in backend
2021-05-19 17:01:34,243 INFO    MainThread:12744 [wandb_run.py:_console_start():1524] atexit reg
2021-05-19 17:01:34,243 INFO    MainThread:12744 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.WRAP
2021-05-19 17:01:34,244 INFO    MainThread:12744 [wandb_run.py:_redirect():1414] Wrapping output streams.
2021-05-19 17:01:34,244 INFO    MainThread:12744 [wandb_run.py:_redirect():1438] Redirects installed.
2021-05-19 17:01:34,244 INFO    MainThread:12744 [wandb_init.py:init():546] run started, returning control to user process
2021-05-19 17:01:34,244 INFO    MainThread:12744 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-19 17:01:34,244 INFO    MainThread:12744 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-19 17:01:37,491 INFO    MainThread:12744 [wandb_init.py:_resume_backend():258] resuming backend
2021-05-19 17:01:39,838 INFO    MainThread:12744 [wandb_watch.py:watch():39] Watching
2021-05-19 17:04:48,185 INFO    MainThread:12744 [wandb_config.py:__setitem__():141] config set save_path = model_save\19_May_202105_04PM_model.pth - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x00000245B9962F88>>
2021-05-19 17:04:48,185 INFO    MainThread:12744 [wandb_run.py:_config_callback():749] config_cb save_path model_save\19_May_202105_04PM_model.pth None
2021-05-19 17:05:05,240 INFO    MainThread:12744 [wandb_init.py:_pause_backend():249] pausing backend
2021-05-19 17:05:05,240 INFO    MainThread:12744 [jupyter.py:save_ipynb():265] not saving jupyter notebook
2021-05-19 17:08:39,865 INFO    MainThread:12744 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 0
2021-05-19 17:08:39,866 INFO    MainThread:12744 [wandb_run.py:_restore():1466] restore
2021-05-19 17:08:40,547 INFO    MainThread:12744 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
  media_count: 95
}
pusher_stats {
  uploaded_bytes: 6324143
  total_bytes: 6324143
}

2021-05-19 17:08:41,191 INFO    MainThread:12744 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 95
}
pusher_stats {
  uploaded_bytes: 6324143
  total_bytes: 6374917
}

2021-05-19 17:08:42,796 INFO    MainThread:12744 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
  media_count: 95
}
pusher_stats {
  uploaded_bytes: 6374917
  total_bytes: 6374917
}

2021-05-19 17:08:42,913 INFO    MainThread:12744 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 95
}
pusher_stats {
  uploaded_bytes: 6374917
  total_bytes: 6374917
}

2021-05-19 17:08:44,066 INFO    MainThread:12744 [wandb_run.py:_show_summary():1771] rendering summary
2021-05-19 17:08:44,067 INFO    MainThread:12744 [wandb_run.py:_show_history():1809] rendering history
2021-05-19 17:08:44,069 INFO    MainThread:12744 [wandb_run.py:_show_files():1838] logging synced files
